### DISCUSSION TITLE
part-ii/2020-120/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#107] [Question comments]
Part II 2020 Probability and Measure: Paper 1, Section II, 27K 

https://questions.tripos.org/part-ii/2020-120/

**Comment by eha1234** [comment]
Any hints for part (d)?

**Comment by ArthurConmy** [comment]
For d), compute

$$ D(f) = \frac{1}{n}  + \frac{1}{n^2} \underbrace{\sum_{i \neq j} \mathbb{ E} f(Z_i) f(Z_j) }_{\text{call this $E$}} $$

Consider $Z_1$ and $Z_2$. Let $\mathcal{F}$ be the set of all functions mentioned in the hint. Then 

$$\sum_{f \in \mathcal{F}} \mathbb{ E} [f(Z_1)f(Z_2)]  \ge \sum_{p, q} \mathbb{ P}[ Z_1 \in I_p , Z_2 \in I_q]  \left( - \frac{k}{2} \right) \ge - \frac{k}{2} \sum_p \mathbb{ P}[ Z_1 \in I_p , Z_2 \not \in I_p]  \ge  - k \times \frac{1}{k} \times \frac{k}{2} = - \frac{k}{2}. $$

So $\sum_{f \in \mathcal{F}} E  \ge \frac{-kn(n-1)}{2}$. This means that there's $f \in \mathcal{F}$ so that $E \ge \frac{-n(n-1)}{2(k-1)}$. But $n$ is fixed and we have the luxury of sending $k \to \infty$ and hence this lower bound approaches 0, hence the result after supping.

**Reply by eha1234** [reply]
Thank you! I agree overall, although confused about some details. On a technical point, I'm assuming all the probabilities without brackets should read $\mathbb{P}(Z_1 \in I_p , Z_2 \in I_q)$ etc. I agree with the third inequality follows, but I'm not sure what $k \times 1/k$ means here? Also, I think there may be a $1/n^2$ in the sum in front of $E$, but this doesn't make a difference to the argument. 

This is neat, actually. Don't think I could have come up with it in an exam lol (although perhaps the summing thing is a standard argument), but really nice.

**Reply by ArthurConmy** [reply]
(edited, thanks) The $k \times \frac{1}{k} \times \frac{k}{2}$ comes from:

1) $\frac{k}{2}$ from that factor outside the sum
2) there are $k$ values of $p$
3) each term in the sum is at most $1/k$ , because the probability has to be less than $\mathbb{P}[ Z_1 \in I_p ]$
